docs/source/configuration_file.rst

```diff
@@ -981,10 +981,24 @@ being used for the segmentation task).
                    "Final activation layer. Options: ``sigmoid`` (default), ``relu``(normalized ReLU), or ``softmax``."
                ]
            },
+           "length_2D": {
+                "type": "[int, int]",
+                "description": "(Optional) Size of the 2D patches used as model's input tensors.",
+                "required": "false"
+            },
+            "stride_2D": {
+                "type": "[int, int]",
+                "$$description": [
+                    "(Optional) Pixels' shift over the input matrix to create 2D patches. Ex: Stride of [1, 2]\n",
```

Comment:
Should we mention "strictly positive integer"? I have not thought much when inputting `[0, 0]` (which is stupid I know) --> gave me an error.

Reply:
Fixed in e42d9dc and e4dacd5.
I'm wondering if I should add length_2D and stride_2D in the doc in this PR or if I should wait for the inference part to be dealt with. With only this PR, the length and stride works only for the training part but the reconstruction of the image will not work yet. What do you think?

Reply:
I think it should be added from now, ie in this PR, as it will be present in the config file etc.